rllib/env/utils.py

```diff
@@ -59,7 +74,11 @@ def try_import_open_spiel(error: bool = False):
         return None
 
 
-def _gym_env_creator(env_context: EnvContext, env_descriptor: str) -> gym.Env:
+def _gym_env_creator(
+    env_context: EnvContext,
```

Comment:
can we update the type? it can be env_descriptor can be str or Type, right?

Reply:
done